src/Lib/Lionk.Core.Razor/Attribute/ComponentView.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright © 2024 Lionk Project
+
+namespace Lionk.Core.Component;
+
+/// <summary>
+/// This attribute is used to specify the view that is used to configure the object.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class)]
+public class ComponentView : Attribute
+{
+    /// <summary>
+    /// Gets the views that are used to configure the objects.
+    /// </summary>
+    public static List<ComponentViewModel> ConfigurationViews { get; } = new();
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ComponentView"/> class.
+    /// </summary>
+    /// <param name="configurableType"> The type of the object that is being configured. </param>
+    /// <param name="configurableView"> The type of the view that is used to configure the object. </param>
+    /// <param name="viewMode"> The view mode that is used to configure the object. </param>
+    public ComponentView(Type configurableType, Type configurableView, ComponentViewMode viewMode) => ConfigurationViews.Add(new ComponentViewModel(configurableType, configurableView, viewMode));
```

Comment:
The parameter configurableView can be deleted as it can be found using this.GetType